src/util/chains.ts

```diff
@@ -1,3 +1,5 @@
+import { Currency, Ether, NativeCurrency, Token } from '@uniswap/sdk-core';
+import { WRAPPED_NATIVE_CURRENCY } from '../../src/providers/token-provider';
```

Comment:
you change this back to eliminate the error. not sure why this would be a problem though

`import { WRAPPED_NATIVE_CURRENCY } from '..';`

Reply:
also `../src` wasn't needed in that string..but still doesn't fix it.